components/NewNFTBook.vue

```diff
@@ -680,7 +596,42 @@ useSeoMeta({
 onMounted(async () => {
   try {
     isLoading.value = true
-    await fetchStripeConnectStatusByWallet(wallet.value)
+
+    if (isEditMode.value) {
+      await fetchStripeConnectStatusByWallet(wallet.value)
+      const classResData: any = await $fetch(`${LIKE_CO_API}/likernft/book/store/${classId.value}`, {
+        headers: {
+          authorization: `Bearer ${token.value}`
+        }
+      })
+      if (classResData) {
```

Comment:
What should happen if `classResData` is not found?

Reply:
If there's no classData, editing the price cannot proceed and should throw an error
[🎨 Enhance error handling for class and edition retrieval](https://github.com/likecoin/nft-book-press/pull/332/commits/8dc7208c0da428dc65495b986abf1fb1fe89709e)